app/matplotlib.ipynb

```diff
@@ -123,21 +170,49 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
     "categories = [\"Category A\", \"Category B\", \"Category C\", \"Category D\"]\n",
     "sales = [350, 480, 210, 590]\n",
     "\n",
-    "# write your code here\n"
+    "df_bar = pd.DataFrame({\n",
+    "    'categories': categories,\n",
+    "    'sales': sales,\n",
+    "})\n",
+    "plt.bar(df_bar['categories'], df_bar['sales'])\n",
```

Comment:
The bar plot is missing a title. You should add a title to the plot using `plt.title('Your Title Here')`.